Probesys/bileto
# new: Post a solution

Related to https://github.com/Probesys/bileto/issues/22

Changes proposed in this pull request:

- Drop field message.is_solution and add a ticket.solution (one-to-one relation to a message)
- Add a checkbox to the ticket editor to mark a message as a solution
- Distinguish visually the "solution" message
- Set color of resolved status to green (as solutions) and new/assigned/in progress to orange

How to test the feature manually:

1. Open a ticket
2. Check the "solution" checkbox, verify the "resolved" status is selected (and the others are disabled)
3. Post the message, verify the status of the ticket is "resolved" and the message is green

Pull request checklist:


_If you think one of the item isn’t applicable to the PR, please check it
anyway and precise `N/A` next to it._
